Packs/TheHiveProject/Integrations/TheHiveProject/TheHiveProject.py

```diff
@@ -427,124 +459,197 @@ def search_cases_command(client: Client, args: dict):
         arguments = args.get('query', None)
     else:
         arguments = {k: True if v == 'true' else v for k, v in args.items() if v is not None}
-        arguments = {k: False if v == 'false' else v for k, v in arguments.items()}
+        arguments = {k: False if v == 'false' else v for k, v in arguments.items() if v is not None}
     try:
         arguments = json.loads(arguments)
     except Exception:
         pass
     cases = client.search_cases(arguments)
-    output_results(
-        title='TheHive Cases search:',
-        headers=['id', 'title', 'description', 'createdAt'],
+    if cases:
+        for case in cases:
+            case_date_dt = dateparser.parse(str(case['createdAt']))
+            if case_date_dt:
+                case['createdAt'] = case_date_dt.strftime(DATE_FORMAT)
+        read = tableToMarkdown('TheHive Cases search:', cases, ['id', 'title', 'description', 'createdAt'])
+    else:
+        read = "No cases were found."
+
+    return CommandResults(
         outputs_prefix='TheHive.Cases',
-        outputs_key_field='id',
-        outputs=cases
+        outputs_key_field="id",
+        outputs=cases,
+        readable_output=read,
     )
 
 
 def update_case_command(client: Client, args: dict):
-    case_id = str(args.get('id', ''))
+    case_id = args.get('id')
 
     # Get the case first
     original_case = client.get_case(case_id)
     if not original_case:
-        return_error(f'Could not find case ID {case_id}')
+        return_error(f'Could not find case ID {case_id}.')
     del args['id']
     for k, v in args.items():
         v = v.split(",") if k in ['tags'] and "," in v else v
         original_case[k] = v
-    res = client.update_case(case_id, args)
-    if type(res) == tuple:
-        return_error(f'Error updating case ({res[0]}) - {res[1]}')
-    output_results(
-        title=f'TheHive Update Case ID {case_id}:',
-        headers=['id', 'title', 'description', 'createdAt'],
+    case = client.update_case(case_id, args)
+    if type(case) == tuple:
```

Comment:
better to use `isinstance`: https://switowski.com/blog/type-vs-isinstance
```suggestion
    if isinstance(case, tuple):
```